www/pages/docs/server-rendering.md

```diff
@@ -217,7 +217,7 @@ console.log(json); // { status: 200, message: 'some data' }
 ```
 
 **Steps**
-1. Make sure you are using Node `v18.12.1`
+1. Make sure you are using Node `v18.15.0`
```

Comment:
Actually, `18.12.1` I think still works?  So maybe this doesn't need to specifically called out 🤷